crates/driver/src/infra/notify/notification.rs

```diff
@@ -18,4 +18,16 @@ pub enum Kind {
     NonBufferableTokensUsed,
     /// Solver don't have enough balance to submit the solution onchain.
     InsufficientBalance,
+    /// Result of winning solver trying to settle the transaction onchain.
+    Settled(SettleKind),
```

Comment:
nit: `Settlement` fits more our new non-composed words naming (otherwise it should be mod settle::Kind)

Reply:
renamed